.circleci/config.yml

```diff
@@ -0,0 +1,170 @@
+                      libgdbm-dev libncurses5-dev automake libtool bison
+      - run:
+          name: virtual-env
+          command: |
+                    if [ ! -d "$HOME/.venv" ]; then
+                      python3 -m venv ~/.venv
+                    fi
+                    echo 'source $HOME/.venv/bin/activate' >> $BASH_ENV
+      - run:
+          name: pre
+          command: |
+                    sudo rm -rf /var/cache/apt/archives
+                    sudo ln -s $HOME/.apt-cache /var/cache/apt/archives
+                    mkdir -p $HOME/.apt-cache/partial
+                    mkdir -p $HOME/.RLibrary
+                    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+                    export "NVM_DIR=$HOME/.nvm"
+                    [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+                    nvm install 6.10.2
+                    nvm alias default node
+                    echo 'source $NVM_DIR/nvm.sh' >> $BASH_ENV
+                    echo 'export PATH=/usr/lib/go-1.9/bin:$PATH' >> $BASH_ENV
+                    echo 'export NVM_DIR=$HOME/.nvm' >> $BASH_ENV
+                    echo 'export CIRCLE_BUILD_IMAGE=ubuntu-12.04' >> $BASH_ENV
+                    echo 'export PATH=/opt/cabal/bin:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/coala-bears/node_modules/.bin:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/coala-bears/vendor/bin:$PATH' >> $BASH_ENV
+                    echo 'export LINTR_COMMENT_BOT=false:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/dart-sdk/bin:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/.cabal/bin:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/infer-linux64-v0.7.0/infer/bin:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/pmd-bin-5.4.1/bin:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/bakalint-0.4.0:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/elm-format-0.18:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/.local/tailor/tailor-latest/bin:$PATH' >> $BASH_ENV
+                    echo 'export PATH=$HOME/phpmd:$PATH' >> $BASH_ENV
+                    echo 'export R_LIB_USER=$HOME/.RLibrary:$PATH' >> $BASH_ENV
+      - run:
+          name: AptDependencies
+          command:  bash .ci/deps.apt.sh
+      - run:
+          name: GeneralDependencies
+          command: |
+                    gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+                    curl -sSL https://get.rvm.io | bash -s stable
+                    source /etc/profile.d/rvm.sh
+                    rvm install 2.2.2
+                    rvm use 2.2.2 --default
+                    ruby -v
+                    gem install bundler
+                    bash .ci/deps.sh
```

Comment:
The proper way to indent is 2 spaces under each block. Fix that in each of the sub blocks.